addons/account/models/account_move.py

```diff
@@ -1378,24 +1379,29 @@ def _compute_partner_credit_warning(self):
                            move.move_type == 'out_invoice' and \
                            move.company_id.account_use_credit_limit
             if show_warning:
-                amount_total_currency = move.currency_id._convert(move.tax_totals['amount_total'], move.company_currency_id, move.company_id, move.date)
-                updated_credit = move.partner_id.commercial_partner_id.credit + amount_total_currency
-                move.partner_credit_warning = self._build_credit_warning_message(move, updated_credit)
+                updated_credit = move.partner_id.commercial_partner_id.credit + move.partner_credit_additional_amount
+                move.partner_credit_warning = self._build_credit_warning_message(
+                    move, updated_credit, move.tax_totals['amount_total'] > 0.0)
 
-    def _build_credit_warning_message(self, record, updated_credit):
-        ''' Build the warning message that will be displayed in a yellow banner on top of the current record
+    def _compute_partner_credit_additional_amount(self):
+        for move in self:
+            move.partner_credit_additional_amount = 0.0
```

Comment:
The point was to not have that weird "additional" thing
```suggestion
            move.partner_credit = move.partner_id.commercial_partner_id.credit
```